drivers/bus/isapnp/hardware.c

```diff
@@ -20,13 +20,16 @@
 
 #endif /* UNIT_TEST */
 
+ULONG_PTR IsaAddressPort = ISAPNP_ADDRESS_PCAT;
+ULONG_PTR IsaWriteDataPort = ISAPNP_WRITE_DATA_PCAT;
```

Comment:
I wonder whether they could be put directly into an array ("IsaWritePorts", ...), something that conveys what they are for) or something like that.
If so, this could allow avoiding the two
```c
    Ports[0] = IsaWriteDataPort;
    Ports[1] = IsaAddressPort;
```
done below :thinking:

Reply:
That's a good suggestion, thanks